sdk/storage/storage-blob-changefeed/review/storage-blob-change-feed.api.md

```diff
@@ -0,0 +1,91 @@
+## API Report File for "@azure/storage-blob-changefeed"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { BlobServiceClient } from '@azure/storage-blob';
+import { PagedAsyncIterableIterator } from '@azure/core-paging';
+
+// @public (undocumented)
+export class BlobChangeFeedClient {
+    constructor(blobServiceClient: BlobServiceClient);
+    // (undocumented)
+    getChanges(options?: ChangeFeedGetChangesOptions): PagedAsyncIterableIterator<BlobChangeFeedEvent, BlobChangeFeedEventPage>;
+    }
+
+// @public (undocumented)
+export interface BlobChangeFeedEvent {
+    // (undocumented)
+    data: BlobChangeFeedEventData;
+    // (undocumented)
+    dataVersion?: string;
+    // (undocumented)
+    eventTime: string;
+    // (undocumented)
+    eventType: BlobChangeFeedEventType;
+    // (undocumented)
+    id: string;
+    // (undocumented)
+    metadataVersion: string;
+    // (undocumented)
+    subject: string;
+    // (undocumented)
+    topic: string;
```

Comment:
What are "subject" and "topic"? These seem like synonyms on the surface.

Reply:
Here is a sample of BlobCreated event.
```
[{
  "topic": "/subscriptions/{subscription-id}/resourceGroups/Storage/providers/Microsoft.Storage/storageAccounts/my-storage-account",
  "subject": "/blobServices/default/containers/test-container/blobs/new-file.txt",
  "eventType": "Microsoft.Storage.BlobCreated",
  "eventTime": "2017-06-26T18:41:00.9584103Z",
  "id": "831e1650-001e-001b-66ab-eeb76e069631",
  "data": {
    "api": "PutBlockList",
    "clientRequestId": "6d79dbfb-0e37-4fc4-981f-442c9ca65760",
    "requestId": "831e1650-001e-001b-66ab-eeb76e000000",
    "eTag": "\"0x8D4BCC2E4835CD0\"",
    "contentType": "text/plain",
    "contentLength": 524288,
    "blobType": "BlockBlob",
    "url": "https://my-storage-account.blob.core.windows.net/testcontainer/new-file.txt",
    "sequencer": "00000000000004420000000000028963",
    "storageDiagnostics": {
      "batchId": "b68529f3-68cd-4744-baa4-3c0498ec19f0"
    }
  },
  "dataVersion": "",
  "metadataVersion": "1"
}]
```